test/utils.spec.js

```diff
@@ -76,18 +76,10 @@ describe('utils', function() {
         'test=1',
         'foo=baz',
         'somethingElse=%3A%3A%22%22%25%26',
-        'nested%5B1%5D=2', // Unencoded: nested[1]=2
-        'nested%5Ba%20n%20o%20t%20h%20e%20r%5D=',
+        'nested[1]=2',
+        'nested[a%20n%20o%20t%20h%20e%20r]=',
       ].join('&'));
     });
-
-    describe('Stripe-specific cases', function() {
-      it('Handles the `expand` array correctly (producing the form `expand[]=_` for each item', function() {
-        expect(decodeURI(utils.stringifyRequestData({
-          expand: ['a', 'foo', 'a.b.c'],
-        }))).to.equal('expand[]=a&expand[]=foo&expand[]=a.b.c');
-      });
-    });
```

Comment:
why remove this test?

Reply:
on that note... we probably _do_ ideally want want `expand` to continue to be `[]` instead of `[0]`, right? Not a big deal, but it is prettier 😄

Reply:
My preference would be to avoid unnecessary special-casing and use the same encoding scheme everywhere. Which is why I removed this test, there's no real need to test "Stripe-specific" stuff since there are no special cases anymore.

Reply:
Fair enough - I assume we have this behavior in our other libs now too, so best to be consistent. 👍